client/extensions/zoninator/components/settings/zones-dashboard/index.jsx

```diff
@@ -4,50 +4,44 @@
 import React, { PropTypes } from 'react';
 import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
-import { find, flowRight, get, times } from 'lodash';
+import { flowRight, times } from 'lodash';
 
 /**
  * Internal dependencies
  */
 import Button from 'components/button';
 import HeaderCake from 'components/header-cake';
 import SectionHeader from 'components/section-header';
-import sectionsModule from 'sections';
 import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
 import QueryZones from '../../data/query-zones';
 import ZoneItem from './zone-item';
 import ZonePlaceholder from './zone-placeholder';
 import { getZones, isRequestingZones } from '../../../state/zones/selectors';
+import { settingsPath } from '../../../app/util';
 
 const placeholderCount = 5;
 
-const ZonesDashboard = ( { isRequesting, siteId, siteSlug, translate, zones } ) => {
-	const sections = sectionsModule.get();
-	const section = find( sections, ( value => value.name === 'zoninator' ) );
-	const settingsPath = get( section, 'settings_path' );
+const ZonesDashboard = ( { isRequesting, siteId, siteSlug, translate, zones } ) => (
+	<div>
+		<QueryZones siteId={ siteId } />
 
-	return (
-		<div>
-			<QueryZones siteId={ siteId } />
+		<HeaderCake backHref={ `/plugins/zoninator/${ siteSlug }` }>
```

Comment:
You could consider using `addSiteFragment` here, but I'll leave that call up to you.